src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -169,4 +181,142 @@ public static String composeClassHeaderEndpointSampleCode(
             ExprStatement.withExpr(initSettingsVarExpr),
             ExprStatement.withExpr(initClientVarExpr)));
   }
+
+  public static String composeRpcMethodHeaderSampleCode(
+      Method method,
```

Comment:
Nit: Consider adjusting the parameter order to be consistent with the chosen order in #576.

Reply:
Sure.